docs/guides/deployment/kubernetes.md

```diff
@@ -0,0 +1,239 @@
+---
+description: Learn how to run flows on Kubernetes using containers
+tags:
+    - kubernetes
+    - containers
+    - orchestration
+    - infrastructure
+    - deployments
+search:
+  boost: 2
+---
+
+# Running flows with Kubernetes
+
+This guide will walk you through running your flows on Kubernetes. One big advantage of Kubernetes is that most major cloud providers have a managed offering. This means you can leverage the ecosystem of your preferred cloud provider while still maintaining portability. Though much of the guide is general to all cloud providers, there differences, especially when it comes to container registries and access management. We'll focus on the big three: Amazon Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS).
```

Comment:
```suggestion
This guide will walk you through running your flows on Kubernetes. Though much of the guide is general to any Kubernetes cluster, there are differences between the managed Kubernetes offerings between cloud providers, especially when it comes to container registries and access management. We'll focus on the big three: Amazon Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE), Azure Kubernetes Service (AKS).
```

Reply:
I think we should assume that the user is already familiar with the advantages/disadvantages around kubernetes.